server/routes/resetPassword.js

```diff
@@ -3,43 +3,56 @@ const router = express.Router();
 const knex = require("../utils/dbConfig");
 const bcrypt = require("bcryptjs");
 const transporter = require("../utils/nodemailer");
+const API_KEY = process.env.API_KEY;
+const DOMAIN = process.env.DOMAIN_NAME;
+
+const formData = require("form-data");
+const Mailgun = require("mailgun.js");
+
+const mailgun = new Mailgun(formData);
```

Comment:
mailgun config seems to be duplicated, can we move this to a file and re-use everywhere?

Reply:
fixed